### ISSUE TITLE
Missing fixedGhostParticles.xml

### ISSUE THREAD

**Issue by Juanlu001** [sanguinariojoe/aquagpusph#4]
Installation of `master` branch fails with this error in the `sudo make install` step after successful compilation:

```
[...]
-- Up-to-date: /usr/share/aquagpusph/resources/Presets/BINoSlip.xml
-- Up-to-date: /usr/share/aquagpusph/resources/Presets/GP.xml
-- Up-to-date: /usr/share/aquagpusph/resources/Presets/GPFreeSlip.xml
-- Up-to-date: /usr/share/aquagpusph/resources/Presets/GPEnergy.xml
CMake Error at resources/Presets/cmake_install.cmake:36 (FILE):
  file INSTALL cannot find
  "/home/juanlu/PFC/aquagpusph/resources/Presets/CMakeFiles/CMakeTmp/fixedGhostParticles.xml".
Call Stack (most recent call first):
  resources/cmake_install.cmake:45 (INCLUDE)
  cmake_install.cmake:39 (INCLUDE)


make: *** [install] Error 1
```

It is indeed not present in the repository, as far as I can tell.

**Comment by Juanlu001** [comment]
The file is present in 76b7a0388abab85 but has been removed since then.

**Comment by sanguinariojoe** [comment]
My Fault! I realized on that bugs when I started the 3.0 branch. I'm fixing that ASAP, however AQUAgpusph can be used without installing that

**Comment by Juanlu001** [comment]
Thanks!

**Comment by sanguinariojoe** [comment]
Fixed in commit 1cf062c
